packages/kbn-dev-utils/src/ci_stats_reporter/ci_stats_test_group_types.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { CiStatsMetadata } from './ci_stats_reporter';
+
+export type CiStatsTestResult = 'fail' | 'pass' | 'skip';
+export type CiStatsTestType =
+  | 'after all hook'
+  | 'after each hook'
+  | 'before all hook'
+  | 'before each hook'
+  | 'test';
+
+export interface CiStatsTestRun {
+  /**
+   * ISO-8601 formatted datetime representing when the tests started running
+   */
+  startTime: string;
+  /**
+   * Duration of the tests in milliseconds
+   */
+  durationMs: number;
+  /**
+   * A sequence number, this is used to order the tests in a specific test run
+   */
+  seq: number;
+  /**
+   * The type of this "test run", usually this is just "test" but when reporting issues in hooks it can be set to the type of hook
+   */
+  type: CiStatsTestType;
+  /**
+   * "fail", "pass" or "skip", the result of the tests
+   */
+  result: CiStatsTestResult;
+  /**
+   * The list of suite names containing this test, the first being the outermost suite
+   */
+  suites: string[];
+  /**
+   * The name of this specific test run
+   */
+  name: string;
```

Comment:
is this the name of the `test`?  I'm not sure what the name of the `test run` is.

Reply:
Yeah, the text you pass to `it(name, ...)`